test/is_palindrome.hpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+
+bool isPalindrome(int x) {
+    if (x<0) return false;
+    std::string Palindrome = std::to_string(x);
+    int i = 0;
+    while (i < Palindrome.size()/2) {
```

Comment:
Usually `length` is recommended for `std::string`.
```suggestion
    while (i < Palindrome.length() / 2) {
```

Reply:
为什么：
Both string::size and string::length are synonyms and return the exact same value.
https://www.cplusplus.com/reference/string/string/length/

Reply:
就是个习惯问题，一般 string 我看到绝大多数地方都会用 length